src/main/entities/DocumentData.js

```diff
@@ -200,7 +211,8 @@ export default class DocumentData {
 
         // https://www.notthewizard.com/2014/06/17/are-files-appends-really-atomic/
         if (content.length >= 1024 && !release) {
-          release = await lockfile.lock(this.filename, lockOptions);
+          console.log('locking file for saving');
```

Comment:
no need to keep this ?

Reply:
oops I literally was in the process of removing those and then my ADHD kicked in and I forgot. Let me clean them up.